### DISCUSSION TITLE
On-Demand Revalidation always timeout on vercel

### DISCUSSION THREAD

**Discussion by desarrolloBREspana** [vercel/next.js#41416] [Help]
Hello

I have the same app in vercel and heroku, developed in Nextjs

When I call On-Demand Revalidation in vercel always return timeout, but when I call the same endPoint in heroku revalidate is Ok in 800ms

Why does vercel always respond with a timeout if it is the same app?

**Comment by wwwdepot** [comment]
Make sure your APIs are publicly reachable & not blocking outside networks. 

See [Check for Upstream Errors](https://vercel.com/guides/what-can-i-do-about-vercel-serverless-functions-timing-out#check-for-upstream-errors) to help narrowing it down if it's a upstream problem.

Lastly, if you think it's a Vercel problem, you should file a ticket there.
